### PR TITLE
Milestone 1: Setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by dqarias**
In this milestone we:

- Set up HTML & CSS linter
- Build the first 2 section of a personal portafolio site
- Use images and backgrounds
- Use Flexbox in all 2 sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portafolio</title>
+    <link rel="stylesheet" href="./css/styles.css">
+</head>
+<body>
+    <header class="header">
+        <span class="header__logo">My Logo</span>
+        <img src="./images/menu.svg" class="header__nav" alt="navigation icon">
+    </header>
```
Comment (codepantha) [inline index.html]: - [ ] Please make the logo a link that navigates back to the homepage. Users naturally expect the logo to take them back to the homepage when they click it 👍 
- [ ] Please enclose your hamburger button in a `<nav>` as it is part of your navigation

File: css/hero.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
+.hero {
+  background-color: #fff;
+  width: 375px;
+  height: 626px;
+  background-image: url("../images/header-shapes\ mobile@2x.svg");
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (codepantha) [inline css/hero.css]: - [ ] Your entire hero section only takes up a middle section of the available space and has spaces on both sides from `425px` 📱 and above. Please make it take up the whole width as shown in the given design
- [ ] Please make same changes to your navigation
### Your Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @dqarias 👋,

## STATUS: REQUESTED CHANGES ♻️ 

Amazing job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🎯 💡 
- Beautiful UI ✅ 
- Meaningful commit messages ✅ 
- Descriptive PR ✅ 
- Linter checks ✅ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] Detailing how users can clone your repo to their local machines

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (dqarias) [reply index.html]: I've Just done it. Thank you for your feedback, Codepantha!

File: css/hero.css (hunk 2 above)
Comment (dqarias) [reply css/hero.css]: I've just done it. Thank you for your help, Codepantha!

Comment (codepantha) [reply css/hero.css]: You're welcome, @dqarias

**Review by prolajumokeoni — APPROVED** [review]
Hi @dqarias, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**